### DISCUSSION TITLE
Tower Defense - Demo 2

### DISCUSSION THREAD

**Discussion by TimWaeckerle** [Freaky2164/TowerDefense#18] [General]
Hi all,

We created our first game menu! We connected the first "Create Game" button to start our demo from our previous post.

![image](https://user-images.githubusercontent.com/97874901/203643910-1074c242-99d8-4400-a3e7-68888cbf885f.png)

Also, we added a sidebar to place tower objects on the map, and we added a financial system. So from now on, if an enemy dies, you will earn money.

![image](https://user-images.githubusercontent.com/97874901/203645892-b9dc1dd7-aae2-43c1-8d7c-aa509d0410f0.png)

Also this week, we agreed on a new design pattern, which means we will recreate all of our designs and update our demo.
We discovered some bugs and will prioritize bug fixes before continuing development on an unstable foundation.

Tasks this week:
Nico: New Map Design
Paul: Financial System
Lukas: Tower Placement
Tim: Start Menu
Johannes: Sidebar

Thanks for your contribution!

Tower Defense Team

**Comment by DanielSchirmer** [comment]
Hello Team Tower Defence,

the game menu looks impressive! Good job! But maybe it would be better to distribute the different buttons across the screen. What other buildings are you planning? How much will the other buildings cost? We hope you can fix your bugs and are curious to see what else we will learn from you in the future!

~ GuessMaster

**Reply by argastle** [reply]
Hello Guessmaster,

the menu is only for a demo if the functionality works, the grafics will be done later. 

We have some designideas for our game and there will be an archertower, missletower and so on. The balancing of the moneysystem will be done later. 

Tower Defense Team
